tests/spread/store/bundle-upload-and-release/task.yaml

```diff
@@ -35,18 +35,23 @@ execute: |
   test -f $BUNDLE_DEFAULT_NAME.zip
 
   # upload
```

Comment:
pedantic: `# upload` -> `# upload and get uploaded revision` to match the other file